bridge/util.go

```diff
@@ -54,13 +54,20 @@ func combineTags(tagParts ...string) []string {
 	return tags
 }
 
-func serviceMetaData(config *dockerapi.Config, port string) (map[string]string, map[string]bool) {
+func serviceMetaData(config *dockerapi.Config, port string, default_attrs string) (map[string]string, map[string]bool) {
 	meta := config.Env
 	for k, v := range config.Labels {
 		meta = append(meta, k+"="+v)
 	}
 	metadata := make(map[string]string)
 	metadataFromPort := make(map[string]bool)
+
+	for _, kv := range strings.Split(default_attrs, ",") {
```

Comment:
In my use case container attrs from env or labels should override the default "forced" attrs.
So I am not sure if the name in the doc make sense, or if container should not override them, meaning that I should move this below.
What do you think?